mis_builder_budget/models/mis_budget_by_account_item.py

```diff
@@ -25,14 +25,11 @@ class MisBudgetByAccountItem(models.Model):
         currency_field="company_currency_id",
     )
     company_id = fields.Many2one(
-        "res.company",
-        related="account_id.company_id",
-        readonly=True,
-        store=True,
+        "res.company", compute="_compute_company_id", readonly=True
```

Comment:
We should declare **`Readonly="True"`** in xml view to avoid data entry problems